src/lib/styles.css

```diff
@@ -1,3 +1,9 @@
+#app {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
Like I've mentioned before, the center style shouldn't appear in our component. The style should be up to the users. In case they really need to align to center. They can simply add the style to the wrapper, rather than doing it in `#app`.

Reply:
got it, will remove it